### ISSUE TITLE
FAQs link in title window does not work

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bliu886/scriptno#151]
```
What steps will reproduce the problem?
1. Click on the ScriptSafe Icon
2. Click on the FAQs link
3. Google servers return NOT FOUND (HTTP 404)

What is the expected output? What do you see instead?
Expected: ScriptSafe FAQ web page
SeeInstead: NOT FOUND (HTTP 404)
What version of the product are you using? On what operating system?
1.0.6.5 On Win64

Please provide any additional information below.


```

Original issue reported on code.google.com by `PudnikAt...@gmail.com` on 29 Dec 2012 at 6:55

**Comment by GoogleCodeExporter** [comment]
```
Thank you! This will be fixed in v1.0.6.9, which will be released in the coming 
days.
```

Original comment by `andr...@gmail.com` on 29 Dec 2012 at 7:34
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
Original comment by `andr...@gmail.com` on 2 Jan 2013 at 2:57
- Changed state: **Fixed**
